molecule/template/molecule.yml.j2

```diff
@@ -1,142 +0,0 @@
----
```

Comment:
I had been treating this file originally as an exhaustive list of all the possible config options to complement the documentation. I wonder if it would be worth updating and placing somewhere?